infra/gcp/clusters/k8s-infra-prow-build-trusted/prow-build-trusted/00-provider.tf

```diff
@@ -0,0 +1,20 @@
+/*
+This file defines:
+- Required Terraform version
+- Required provider versions
+- Storage backend details
+*/
+
+terraform {
+  required_version = ">= 0.12.8"
```

Comment:
Can you bump the version to `0.12.20` ? There is a Lot of bug fixes released.

Reply:
Sounds good, it looks like I've been running `0.12.24` locally FWIW.

Given https://www.terraform.io/upgrade-guides/index.html I'm going to switch this to `~>` instead of `>=` in case there are breaking changes needed to upgrade to `0.13`